lib/SelectionModal/SelectionModalCategory.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { bool, string, number } from 'prop-types';
+import cx from 'classnames';
+
+function SelectionModalCategory({
+  isSelected,
+  className,
+  name,
+  subText,
+  counter,
+  ...rest
+}) {
+  const buttonClasses = cx(
+    `bg-white relative border-solid flex w-full rounded outline-none px-3 py-2 border border-neutral-90 items-center ${className}`,
+    {
+      'text-input-border-focus': isSelected,
+      'hover:border-blue-primary': !isSelected
+    }
+  );
```

Comment:
🚩 At this point it's probably worth going back to cheeky Peaky and seeing if this button type is going to be used elsewhere, and if so get it into GCDS and a new button type. It looks pretty useful to me!

Reply:
This is another weird one, this is as much a new button type as the links in the finder panel sidebar is. Basically I don't think this is a new button type, more its own slightly more complex new thing. 

If it ends up being needed elsewhere then it would be pretty trivial to split out when needed, then we can decide where it lives/what its called based on both usecases.

Reply:
Sounds good - let's keep an eye out for future uses 👀